### PR TITLE
New Install Docs + Common Error

### PR DESCRIPTION
**Pull request by ddavila54**
Ticket to update installation and uninstallation procedure documentation. Also improve documentation to assist with training clarification.

### CODE REVIEW

File: sphinx-docs/Common-problems.md [hunk 1]
```diff
@@ -23,4 +23,19 @@ Common problems
...
+2. Look at each ability command. If there is a variable inside - shown by #{} syntax - the ability will need to be ["unlocked" by another ability](What-is-an-ability.md), in a prior phase, before it can run. 
+
+### I'm getting this go error when I run my server!
+```
+can't load package: package github.com/google/go-github/github: cannot find package "github.com/google/go-github/github" in any of:
+	/usr/local/go/src/github.com/google/go-github/github (from $GOROOT)
+	/home/debian/go/src/github.com/google/go-github/github (from $GOPATH)
+```
+
+1. Check to see of go is properly installed on your system.
```
Comment (wbooth) [inline sphinx-docs/Common-problems.md:35]: sp

File: sphinx-docs/Common-problems.md [hunk 2]
```diff
@@ -23,4 +23,19 @@ Common problems
...
+2. Make sure the go environment variables are properly set.
```
Comment (wbooth) [inline sphinx-docs/Common-problems.md:36]: how do we do this?

File: sphinx-docs/Common-problems.md [hunk 3]
```diff
@@ -23,4 +23,25 @@ Common problems
 ### I ran an adversary and it didn't do everything!
 
 1. Check each ability on the adversary profile. It should show an icon for which operating system it runs on. Match this up with the operating systems of your agents. These are the only abilities an operation will attempt to run.
-2. Look at each ability command. If there is a variable inside - shown by #{} syntax - the ability will need to be ["unlocked" by another ability](What-is-an-ability.md), in a prior phase, before it can run. 
\ No newline at end of file
+2. Look at each ability command. If there is a variable inside - shown by #{} syntax - the ability will need to be ["unlocked" by another ability](What-is-an-ability.md), in a prior phase, before it can run.
```
Comment (wbooth) [inline sphinx-docs/Common-problems.md:26]: Change phase to step

File: sphinx-docs/Plugin-library.md [hunk 4]
```diff
@@ -4,6 +4,10 @@ Plugin library
 Here you'll get a run-down of all open-source plugins, all of which can be found in the plugins/ directory as separate 
 GIT repositories. 
 
+To enable a plugin, add it to the `default.yml` file in the `conf/` directory.
```
Comment (wbooth) [inline sphinx-docs/Plugin-library.md:7]: You might want to add that it should be done when the server is stopped

File: sphinx-docs/Plugin-library.md [hunk 5]
```diff
@@ -168,6 +174,8 @@ The Atomic plugin imports all Red Canary Atomic tests from their open-source Git
 The GameBoard plugin allows you to monitor both red-and-blue team operations. The game tracks points for both sides
 and determines which one is "winning". 
 
+To begin a gameboard exercise, first log in as blue user and begin an operation. Then log in as red user and begin another operation. Open up the gameboard plugin from the GUI and select these new respective red and blue operations to monitor points for each operation.
```
Comment (wbooth) [inline sphinx-docs/Plugin-library.md:177]: Auto-collect operation will be executed automatically if you start a blue agent. Though you can login as blue and start an op simply running a blue agent is sufficient

**Comment by ddavila54** [comment]
Changes made to plugin-library.md. Separate PR for common-problems.md

**Pull request merged**